docs/uninstall-an-operator.md

```diff
@@ -0,0 +1,36 @@
+# How to uninstall an Operator?
+
+In order to uninstall an operator, you need to delete the following resources:
+
+- Subscription
+- ClusterServiceVersion (CSV)
+
+Both `Subscription` and `CSV` are namespaced objects meaning you need to delete a `Subscription` and a `CSV` in a specific namespace where you install the operator into.
+
+## Delete a Subscription
+
+If you wish to look up a list of `Subscription` in a specific namespace to see which `Subscription` you want to delete, you can use the following `kubectl` command:
+```bash
+$ kubectl get subscription -n <namespace>
+```
+
+You can delete the `Subscription` in the namespace that the operator was installed into using this command:
+```bash
+$ kubectl delete subscription <subscription-name> -n <namespace>
+```
```

Comment:
I think it would be nice to include docs on deleting _both_ subscription and csv.

```
$ CSV=kubectl delete subscription <subscription-name> -n <namespace> -o json | jq '.status.installedCSV'
$ kubectl delete subscription <subscription-name> -n <namespace>
$ kubectl delete csv $CSV -n <namespace>
```